client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx

```diff
@@ -133,239 +127,236 @@ const InvestigatedPersonInfo = (props: Props) => {
     };
 
     return (
-        <Paper className={classes.paper}>
```

Comment:
I need to add this  line       <Paper className={classes.paper}>
because i used it to align the items